scripts/ci.sh

```diff
@@ -1,6 +1,25 @@
 # This script contains commands to be executed by the CI tool.
 NPROC=$(nproc)
 
+update_go() {
+	echo "Update go version..."
```

Comment:
You can add check if the current version is 1.13 or later so that we can skip this step, which will speed up the CI.

Reply:
Added version comparasion before upgrade on CI